x-pack/platform/plugins/shared/stack_connectors/server/connector_types/gemini/gemini.ts

```diff
@@ -380,11 +391,13 @@ export class GeminiConnector extends SubActionConnector<Config, Secrets> {
 
 /** Format the json body to meet Gemini payload requirements */
 const formatGeminiPayload = ({
+  maxOutputTokens,
   messages,
   systemInstruction,
   temperature,
   toolConfig,
 }: {
+  maxOutputTokens?: number | undefined;
```

Comment:
Can this be `maxOutputTokens?: number` ?

Reply:
some copy pasta 🍝 there, thank you